system/objects/types.rst

```diff
@@ -224,8 +226,8 @@ actual URL of the field - e.g. ``https://zammad.com``.
 How does this work...?!
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-As an example, let's say you have an object called ``amazingobject`` - you want
-to open a google search directly with the input from that field.
+As an example, let's say you have an attribute called ``amazingobject`` - you
```

Comment:
Should we also adjust the name of the example attribute? It's only logical, since it will only be saving a reference to another object anyway. Something along the same lines like `amazing_attribute` or whatever would do.

Reply:
It makes sense, yes! I changed it to `amazing_attribute`. However, I noticed the "old" `amazingobject` in the GIF. I deleted it and inserted a short summary of the result.